JS-Algorithms/challenges.js

```diff
@@ -70,7 +98,22 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  const power = (base, exponent) => {
+    let result = 1n;
+    for (let i = 0n; i < exponent; i++) {
+      result *= base;
+    }
+    return result;
+  };
```

Comment:
This function can live outside of ownPower since dont need to be defined each time the ownPower is being called